docs/StardustDocs/topics/schemas.md

```diff
@@ -11,7 +11,7 @@ It ignores order of columns in [`DataFrame`](DataFrame.md), but tracks column hi
 
 In Jupyter environment compile-time [`DataFrame`](DataFrame.md) schema is synchronized with real-time data after every cell execution.
 
-In IDEA projects you can use [gradle plugin](installation.md#data-schema-preprocessor) to extract schema from dataset
+In IDEA projects, you can use [Gradle plugin](gradle.md#configuration) to extract schema from dataset
```

Comment:
*the Gradle plugin
*to extract a schema from the dataset (or schemas from datasets)